src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/combat/killaura/features/KillAuraVelocityHit.kt

```diff
@@ -0,0 +1,123 @@
+package net.ccbluex.liquidbounce.features.module.modules.combat.killaura.features
+
+import net.ccbluex.liquidbounce.config.types.ToggleableConfigurable
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.events.TransferOrigin
+import net.ccbluex.liquidbounce.event.sequenceHandler
+import net.ccbluex.liquidbounce.event.tickHandler
+import net.ccbluex.liquidbounce.features.module.modules.combat.killaura.ModuleKillAura
+import net.ccbluex.liquidbounce.features.module.modules.misc.ModulePacketLogger
+import net.ccbluex.liquidbounce.utils.client.Chronometer
+import net.ccbluex.liquidbounce.utils.client.PacketQueueManager
+import net.ccbluex.liquidbounce.utils.client.chat
+import net.minecraft.entity.LivingEntity
+import net.minecraft.network.packet.c2s.play.ClientCommandC2SPacket
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+import net.minecraft.network.packet.s2c.play.*
+
+object KillAuraVelocityHit : ToggleableConfigurable(ModuleKillAura, "VelocityHit", false) {
+
+    private var considerVelocityHit = false;
+    private var damageReceived = false;
+    private var onGroundTicks = 0;
+    private var isPossible = false;
+
+    val extensionRange by float("Extension", 1.0f, 0.1f..2.0f)
+    val onLag by boolean("OnlyWhileLagging", false)
+
+    private var timer: Chronometer = Chronometer()
+    private var lastPacketTime: ArrayList<Long> = ArrayList()
+
+    private val sampleSize: Int = 10
```

Comment:
```suggestion
    const val sampleSize = 10
```
`sampleSize` can be a `const val`. Will also make detekt less angry. Removed the type annotation since Kotlin can figure out that it's a `Int`.